web/lti.go

```diff
@@ -61,16 +62,19 @@ func loginWithLTI(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Logout(w, r)
 
 		// Don't redirect to signup page if BuildUser is going to fail
-		if user, err = lms.BuildUser(launchData, ""); err != nil {
+		user, err := c.App.BuildNewUserFromLTILaunchData(lms, launchData, "")
+		if err != nil {
 			c.Err = err
 			return
 		}
+		launchData["lis_person_sourcedid"] = user.Username
```

Comment:
It seems the normal style is to have a blank line after the end `}` of an if block.
```suggestion

		launchData["lis_person_sourcedid"] = user.Username
```